dcos/config.py

```diff
@@ -173,6 +173,15 @@ def get_config_val_envvar(name, config=None):
     if config is None:
         config = get_config()
 
+    # This is a list of configs which cannot be overriden by env variables
+    env_blacklist = [
+        'core.dcos_url',
+        'core.dcos_acs_token',
+        'core.mesos_master_url',
+    ]
+    if name in env_blacklist:
+        return config.get(name), None
```

Comment:
As we're automatically migrating to the clusters dir structure and ignoring `DCOS_CONFIG`, should we also ignore `DCOS_URL` and `DCOS_ACS_TOKEN` env variables? As the `dcos cluster ...` commands should be the way to go to manage different clusters.

I'm not sure about `DCOS_MASTER_MESOS_URL` though.